CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 # Version 0.0.0 (YYYY-MM-DD)
+### Changed
+* `ModelRun.get_config()`
+    * Modifies get_config to return un-nested Model Run config
```

Comment:
Maybe change the wording to something like:

`get_config` now returns un-nested Model Run config

also question, what is nested vs. un-nested?

Reply:
hm, cause basically we were returning the config object under the key 'trainingData' in the return object. Karen asked to just return the config object directly.

Reply:
Gotcha, i'm ok with these changes then